gallery/src/pages/StylingComponents.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import {GalleryTile} from '../gallery-components/GalleryTiles';
+
+const StylingRSCPage = () =>
+  <GalleryTile title="How to style your app when using RSC">
+    <p className="nx-p">
+      If you have styling needs that aren't covered by the styles included in React Shared Components (RSC),
+      then you're going to have to create your own styles.
+    </p>
+    <h2 className="h2">Page specific stylesheets</h2>
+    <p className="nx-p">
+      Take a look at <code className="nx-code">frontend/src/main/scss/app-styles.scss</code>. This file
+      contains custom styles for the app and imports external (page specific) stylesheets. Why have page
+      specific stylesheets? By grouping our styles together by use and (ideally) locating the SCSS
+      next to the HTML that it modifies we decrease clutter/noise in our main stylesheet, keep our styles focused
+      to what they modify, and also help maintain a clear separation between code, layout, and styling.
```

Comment:
I think in addition to page-specific styles, component-specific styles should also be mentioned, as they are perhaps even more important.

Reply:
> I think in addition to page-specific styles, component-specific styles should also be mentioned, as they are perhaps even more important.

I added a sub-sub-section to specifically call out component specific styling: https://github.com/sonatype/sonatype-react-shared-components/pull/72/commits/d909ee05869ba31c19106ddbff91c3d287cefaea

Reply:
> I added a sub-sub-section to specifically call out component specific styling: d909ee0

That's actually not what I was talking about.  I meant when an app has its own custom components and needs to style them.